src/myapp.h

```diff
@@ -117,14 +122,11 @@ struct MyApp : wxApp {
         }
 
         if (single_instance) {
-            instance_checker = new wxSingleInstanceChecker(

```

Comment:
you have to be a bit careful with moving the initialization.. like if `single_instance` is false, this init isn't needed to run.

Reply:
Thanks for your review! I now revised the commit.
There is still a unique_ptr, but the managed object will be set to the wxSingleInstanceChecker object on the heap only if single_instance evaluates to true.
Is this ok?